src/application/polymarket/widgets/market-widget.tsx

```diff
@@ -209,8 +269,9 @@ export const Market = ({
                               <InputBase.Label label="Amount" />
                               <Chip className="bg-[#2C3F4F]">
                                 Balance $
-                                {user.safeWalletDetails?.balance.toFixed(2) ??
-                                  0}
+                                {(user.safeWalletDetails?.balance ?? 0).toFixed(
+                                  2,
+                                ) ?? 0}
```

Comment:
I think we can remove this fallback as we have one in the toFixed source

Reply:
good catch!